blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -103,45 +108,44 @@ export class Activity extends Component {
         return (<main>
             <article className="activity">
                 { showRunButton &&
-                <RunButton
-                  runnable={pipeline}
-                  latestRun={latestRun}
-                  buttonType="run-only"
-                  onNavigation={onNavigation}
-                  innerButtonClasses="btn-secondary"
-                />
+                    <ParametersRunButton
```

Comment:
So this contains the regular run button if I read it correctly. 
This is maybe a little bit confusing name wise - it probably still should be "RunButton" but then it has components in it for normal and parametrised run button I think.